tests/dbshell/test_postgresql_psycopg2.py

```diff
@@ -99,3 +100,16 @@ def test_accent(self):
                 pgpass_string,
             )
         )
+
+    @removeHandler
+    def test_sigint(self):
+
```

Comment:
I think a comment would be useful here explaining why the desired behaviour is not to raise a `KeyboardInterrupt`.